bin/src/mock_prove.rs

```diff
@@ -0,0 +1,66 @@
+use ethers_providers::{Http, Provider};
+use std::env;
+use types::eth::BlockTrace;
+use zkevm::circuit::SuperCircuit;
+use zkevm::prover::Prover;
+
+const DEFAULT_BEGIN_BATCH: i64 = 1;
+const DEFAULT_END_BATCH: i64 = i64::MAX;
+
+#[tokio::main]
+async fn main() {
+    log::info!("mock_prove: begin");
```

Comment:
the log::info before env_logger::init() will not print out?  Either println or put it after  env_logger::init();

Reply:
Moved this log after initializing `dotenv` and `env_logger` in https://github.com/scroll-tech/scroll-zkevm/pull/99/commits/dfc9b749e3a795d2eee24f4f321dcbfbff702c30.